libs/domain/site/mode-selector/src/mode-selector.def.ts

```diff
@@ -0,0 +1,11 @@
+import { componentDef } from '@spryker-oryx/core';
+
+export const siteModeSelectorComponent = componentDef({
+  name: 'oryx-site-mode-selector',
```

Comment:
```suggestion
  name: 'oryx-site-color-mode-selector',
```

Think we should rename the feature and all files, names, etc. to include color.